test/Fail/Issue2640.err

```diff
@@ -1,11 +1,14 @@
 Failed to solve the following constraints:
   tt = _b_38
     : (λ { true → ⊤ ; false → ⊥ }) (test true (c (_b_21 (n = true))))
+    (blocked on any(_33, _35))
   ⊥ =< (λ { true → ⊤ ; false → ⊥ })
        (test false (c (_b_21 (n = false))))
+    (blocked on _23)
   ⊤ =< (λ { true → ⊤ ; false → ⊥ })
        (test true (c (_b_21 (n = true))))
-  _b_21 = n : Bool
+    (blocked on _23)
+  _b_21 = n : Bool (blocked on _b_21)
```

Comment:
This doesn't make sense, how can the equation `_b_21 = n` be blocked on `_b_21` itself?

Reply:
I opted to block failed meta instantiations on the meta (among other things). If the meta gets solved we do want to revisit this constraint.